src/p_saveg.c

```diff
@@ -76,6 +79,35 @@ inline static void saveg_write_pad(void)
     }
 }
 
+typedef union
+{
+    byte b[4];
+    float f;
+} fdata_t;
+
+inline static void saveg_write_float(float value)
+{
+    fdata_t data = {.f = SDL_SwapFloatLE(value)};
+
+    saveg_buffer_size(4);
+
+    for (int i = 0; i < 4; i++)
+    {
+        savep_putbyte(data.b[i]);
+    }
+}
+
+inline static float saveg_read_float(void)
+{
+    fdata_t data;
+
+    for (int i = 0; i < 4; i++)
+    {
+        data.b[i] = saveg_read8();
+    }
+
+    return SDL_SwapFloatLE(data.f);
```

Comment:
We have the m_swap.h file, so maybe we should add another macro, such as `SWAP_FLOAT`, for consistency.

Reply:
Fixed here: https://github.com/fabiangreffrath/woof/pull/2280/commits/7791920d41db78224372b940d3df35b9642342e8